Jellyfin.Server/Program.cs

```diff
@@ -260,13 +260,20 @@ private static IWebHostBuilder CreateWebHostBuilder(ApplicationHost appHost, ISe
                         }
                     }
                 })
-                .UseContentRoot(appHost.ContentRoot)
                 .ConfigureServices(services =>
                 {
                     // Merge the external ServiceCollection into ASP.NET DI
                     services.TryAdd(serviceCollection);
                 })
                 .UseStartup<Startup>();
+
+            // Set the root directory for static content, if one exists
+            if (!string.IsNullOrEmpty(appHost.ContentRoot))
```

Comment:
why not use `appHost.IsHostingContent`?

Reply:
Good point, I've changed it to use `IsHostingContent`